source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,348 @@
+your App's backend. You should be aware of the side effects and plan ahead.
+
+- Client apps that use an SDK with with a realm-core version lower than TBD will
+  not be able to sync after migration.
+- How long a migration takes is directly proportional to the amount of data that
+  needs to be migrated. The more data, the longer it will take to migrate.
+- Migrations can be :ref:`canceled <realm-sync-migrate-cancel>` during the 
+  migration or :ref:`reverted <realm-sync-migrate-revert>` within a certain 
+  timeframe after the migration is finished.
+- Progress notifications will not work after migration because Flexible Sync does
+  not support them.  
+- Client apps that were connected to the Partition-Based Sync backend will 
+  experience a client reset after the backend is migrated to Flexible Sync. You 
+  should use a :ref:`client reset <handle-a-client-reset>` handler with recovery enabled.
+- Creating new tables in a migrated Flexible Sync backend using 
+  :ref:`Development Mode <development-mode>` from a Partition-Based Sync client
+  may lead to unexpected behavior.
+
+.. _realm-sync-migrate-stages:
+
+Migration Stages
+~~~~~~~~~~~~~~~~
+
+There are three stages to a Sync mode migration. You can see your migration's
+stage in the App Services UI.
+
+#. **Syncing Partition-Based Sync metadata** 
+   
+   At this stage, client apps can connect to your App's backend, but local writes 
+   will not sync. These writes will be lost if you don't have a 
+   :ref:`client reset handler <handle-a-client-reset>` to recover unsynced changes.
+	 
+   Clients can also receive updates to synced collections, but they won't
+   be able to receive upload acknowledgements.
+
+#. **Building Flexible Sync metadata** 
+   
+   Clients cannot connect at this stage. The duration of this stage is directly
+   proportional to the amount of data in synced collections.
+
+#. **Completed migration** 
+
+   :guilabel:`Manage Sync Migration` appears on the Device Sync page if the 
+   migration hasn't been committed or reverted.
+
+After the migration has finished, check out the :ref:`After Migration
+<realm-sync-migrate-after>` section for more information about how
+Partition-Based Sync clients interact with the Flexible Sync backend.
+
+Migrate Partition-Based Sync to Flexible Sync
+---------------------------------------------
+
+After beginning migration from Partition-Based Sync to Flexible Sync, the process
+is mostly automatic. However, you can control important parts of the migration.
+
+During a migration, you can cancel it. After a migration, you can revert it 
+or manually commit the migration.
+
+Begin Migration
+~~~~~~~~~~~~~~~
+
+Before you begin a migration, make sure you're aware of the
+:ref:`migration stages <realm-sync-migrate-stages>` and
+:ref:`effects <realm-sync-migrate-what-to-know>`.
+
+During migration, client apps may experience side effects, most of which are similar
```

Comment:
"Side effects may include painful rash..." ;-) 
Seriously, I always think of "side effects" in that way, and I wonder if others do, too? Also, do client apps "experience" anything? (not to be too philosophical here...)

If so, consider "client apps may be impacted similar to what happens when you terminate (etc)"

Reply:
Ah, there I go anthropomorphizing technology again. I do hope every realm has a beautiful experience though. 😄 

I'll re-word this.